windows.md

```diff
@@ -390,6 +390,49 @@ With those compatibility things out of the way, you're ready to start the system
     4. Open the start menu and search for "Ubuntu". Start it - it should ask you to create a user with a password. This will be your user to log in to your Ubuntu Linux Subsystem - note down the username and password somewhere secure to make sure you do not forget it.
     5. Open the start menu and search for "Docker Desktop". Start it and go to the Settings. Under the General tab, you will find an option called "Use WSL 2 based engine". Make sure this is checked.
 
+    If opening Docker Desktop from **Option A or B** shows an error, you may need to enable virtualization on your machine
+
+    The error might look like this:<br><br>
+    <img src="docker-desktop-opening-error.avif"><br><br>
+
+    Close PowerShell and open it again as administrator (like in step 2)
```

Comment:
https://github.com/upleveled/system-setup/pull/16/commits/0db74a60c1f8c953e516bf024c0f2c568e7bf199

Reply:
what I mean is, it should be step number 1, where the current number 1 is `Restart your computer` (this should become number 2)

Reply:
b892a20224774ac63333df78e3df02633c1ea849

Reply:
> Hmmm, it seems wrong because when they restart their computer, they will lose the PowerShell instance.

the point is not about grouping the steps because it's during a single power cycle - it's about grouping them so that students know where the steps start and end, to make it easier to skip the whole thing, and clear that they are now in the instructions for trying to deal with the Docker error

Reply:
bfb2c747b5f11179451e3f22e6c9344e007956d0